zrml/prediction-markets/src/lib.rs

```diff
@@ -443,11 +445,27 @@ mod pallet {
                     Error::<T>::MarketEditRequestAlreadyInProgress
                 );
                 m.status = new_status;
+
+                if m.is_redeemable() {
+                    for outcome in m.outcome_assets(market_id) {
+                        let admin = Self::market_account(market_id);
+                        let is_sufficient = true;
+                        let min_balance = 1u8;
+                        T::AssetCreator::create(
+                            outcome.into(),
+                            admin,
```

Comment:
Do we expect to use the admin team of the outcome assets? Wonder how much bearing this has or if the main goal is just to have an addy that's not used anywhere else?

Reply:
The goal is just to add an address that is not used anywhere else. Admin functions should not be invoked on `MarketAssets`.